index.html

```diff
@@ -5,62 +5,160 @@
+                    <h6>balerum <br>1,000 000 Views</h6>
+                </div>
+
+            </div>
+            <div class="flex videos-spacing">
+
+                <figure><img src="./Images/thumbnail5.png" alt="Porch pirate vs. glitter bomb"></figure>
+
+                <div>
+                    <p>Porch pirate vs. glitter bomb</p>
+                    <h6>balerum <br>1,000 000 Views</h6>
+                </div>
+            </div>
+            <div class="flex videos-spacing">
+
+                <figure><img src="./Images/thumbnail6.png" alt="Porch pirate vs. glitter bomb"></figure>
+                <div>
+                    <p>Playing flappy bird, every like makes it faster (world record..)</p>
+                    <h6>balerum <br>1,000 000 Views</h6>
+                </div>
+            </div>
+            <div class="flex videos-spacing">
+
+                <figure><img src="./Images/thumbnail7.png" alt="
+                    Eat in one bite challenge | Tacos, Burritos (People Vs)"></figure>
+                <div>
+                    <p>Eat in one bite challenge | Tacos, Burritos (People Vs)</p>
+                    <h6>balerum <br>1,000 000 Views</h6>
+                </div>
+            </div>
+        </section>
+
+    </div>
+
+    <section class="channel-info margin">
 
-    <section class="description margin">
         <h3>
             Baby Shark Dance and more | Summer Songs Special | +Compilation | Pinkfong Songs for Children
         </h3>
         <div class="flexbox">
-            <img class="margin_between" src="/Images/uploader.PNG" alt="">
+            <img class="margin_between" src="./Images/uploader.PNG" alt="User">
             <div>
-             <p class="margin_between">
-              Peppa pig in English
-             </p>
-             <div class="flexbox margin_between">
-                <button class="suscriptor_button"><img class="white_icon" src="/Images/subscribe_button.PNG" alt=""> Suscribe</button>
-                <p class="suscriptor_number">799</p>
-                <p class="views">
-                    50,016 views
+                <p class="margin_between">
+                    Peppa pig in English
                 </p>
-             </div>
-             
+                <div class="flexbox margin_between">
+                    <button class="suscriptor_button"> <img class="white_icon" src="./Images/subscribe_button.PNG" alt="youtube small-icon">
+                     Suscribe</button>
+                    <p class="suscriptor_number">799</p>
+                    <p class="views">
+                        50,016 views
+                    </p>
+                </div>
+
+            </div>
+
+        </div>
+
+        <div class="flexbox gray_border_top">
+            <img class="small_icons_square" src="./Images/add_icon.PNG" alt="add icon">
+            <p class="aling">Add to</p>
+            <img class="small_icons" src="./Images/share_icon1.png" alt="share icon">
```

Comment:
This icon is not loading as well. Kindly check what is happening.

Reply:
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/61803988/81373817-4478a080-90c3-11ea-9a3d-5cac2714fc65.gif)

Hi Lucas, thanks for the code review, we have made the required changes!